index.html

```diff
@@ -339,128 +325,119 @@ <h1>Important Terms</h1>
 			</dd>
-			<dt><dfn data-export="" data-lt="state|states">State</dfn></dt>
+			<dt><dfn data-lt="state|states">State</dfn></dt>
 			<dd>
 			  <p>A state is a dynamic <a class="termref" href="#dfn-property">property</a> expressing characteristics of an <a>object</a> that may change in response to user action or automated processes. States do not affect the essential nature of the object, but represent data associated with the object or user interaction possibilities. See <a class="specref" href="#statevsprop">clarification of states versus properties</a>.</p>
 			</dd>
-			<dt><dfn data-export="">Sub-document</dfn></dt><!--Not used-->
+			<dt><dfn>Sub-document</dfn></dt><!--Not used-->
 			<dd>
 			  <p>Any document created from a <code>&lt;frame&gt;</code>, <code>&lt;iframe&gt;</code> or similar mechanism. A sub-document may contain a document, an application or any widget such as a calendar pulled in from another server. In the <a class="termref">accessibility tree</a> there are two <a class="termref" data-lt="accessible object">accessible objects</a> for this situation&mdash;one represents the <code>&lt;frame&gt;</code>/<code>&lt;iframe&gt;</code> element in the parent document, which parents a single <a>accessible object</a> child representing the spawned document contents.</p>
 			</dd>
-			<dt><dfn data-export="">Target Element</dfn></dt>
+			<dt><dfn>Target Element</dfn></dt>
 			<dd>
 			  <p>An element specified in a <abbr title="Accessible Rich Internet Application">WAI-ARIA</abbr> relation. For example, in <code> &lt;div aria-controls=”elem1”&gt;</code>, where <code>“elem1”</code> is the ID for the target element.</p>
 			</dd>
-			<dt><dfn data-export="">Taxonomy</dfn></dt><!--Not used-->
+			<dt><dfn>Taxonomy</dfn></dt><!--Not used-->
 			<dd>
-			  <p>A hierarchical definition of how the characteristics of various <a class="termref" data-lt="class">classes</a> relate to each other, in which classes inherit the properties of superclasses in the hierarchy. A taxonomy can comprise part of the formal definition of an <a>ontology</a>.</p>
+			  <p>A hierarchical definition of how the characteristics of various classes relate to each other, in which classes inherit the properties of superclasses in the hierarchy. A taxonomy can comprise part of the formal definition of an <a>ontology</a>.</p>
 			</dd>
-			<dt><dfn data-export="">Text node</dfn></dt>
+			<dt><dfn>Text node</dfn></dt>
 			<dd>
 			  <p>Type of <abbr title="Document Object Model">DOM</abbr> <a class="termref" data-lt="node">node</a> that represents the textual content of an <a class="termref">attribute</a> or an <a class="termref">element</a>. A Text node has no child nodes.  </p>
 			</dd>
-			<dt><dfn data-export="">Tooltip attribute</dfn></dt><!--Accname-->
+			<dt><dfn>Tooltip attribute</dfn></dt><!--Accname-->
```

Comment:
need to move this to accname - then no export needed

Reply:
Ok, I wasn't sure where to put it in Accname. Could you send a PR for it?